src/config/ConfigManager.cpp

```diff
@@ -147,6 +147,7 @@ void CConfigManager::setDefaultVars() {
     configValues["input:touchpad:tap-to-click"].intValue = 1;
     configValues["input:touchpad:drag_lock"].intValue = 0;
     configValues["input:touchpad:scroll_factor"].floatValue = 1.f;
+    configValues["input:touchdevice:td_rotation"].intValue = 0;
```

Comment:
can't we just autodetect it based on the output config?

Reply:
I couldn't find a link between monitor and touch screen. It could maybe be in the config file, but I'm not sure it's the best idea, considering that some touch devices don't have screens.

Reply:
touch devices always have screens. see `wlr_touch_from_input_device(touchdev->pwlrdevice)->output`

Reply:
Not always. Unfortunately null pointers are a thing, and at least in my case the touch device is not linked to the screen:

```
        auto output = wlr_touch_from_input_device(PTOUCHDEV->pWlrDevice)->output_name;
        Debug::log(LOG, "[TEST] Output screen: %s", output);
        Debug::log(LOG, "[TEST] Output screen address: %i", output);
```

logs:

```
[LOG] [TEST] Output screen: (null)
[LOG] [TEST] Output screen address: 0
```

Reply:
ay that fucking sucks. Alright then. just change the name, drop `td_` cuz you already put it in `touchdevice`

Reply:
If you're ok with it I'll change it to `transform`, I added the matrices missing to match the 7 possible arguments of `monitor=NAME,transform,TRANSFORM`.

Reply:
feel free. sounds good